k8s/util.py

```diff
@@ -0,0 +1,80 @@
+import logging
```

Comment:
Lets rename this to deprecationchecker.py or something, not util.py

Reply:
ok